cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/BrokerState.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class BrokerState {
```

Comment:
Add Javadoc to explain what this class is for. E.g. "Java representation of the JSON response from the /v1/broker-state endpoint of the KafkaAgent"